code/controllers/subsystem/non_firing/persistent_data.dm

```diff
@@ -13,17 +13,26 @@ SUBSYSTEM_DEF(persistent_data)
 	flags = SS_NO_FIRE
 	/// List of atoms registered into the subsystem for persistent data storage. Can be anything at all
 	var/list/registered_atoms = list()
+	/// Set to true after a the end of the round to prevent griefing being saved
+	var/end_of_round_shutdown = FALSE
 
 /datum/controller/subsystem/persistent_data/Initialize()
 	// Load all the data of registered atoms
 	for(var/atom/A in registered_atoms)
 		A.persistent_load()
 
 /datum/controller/subsystem/persistent_data/Shutdown()
+	// Stops data from being saved twice on round end
+	if(end_of_round_shutdown)
+		return
 	// Save all the data of registered atoms
 	for(var/atom/A in registered_atoms)
 		A.persistent_save()
 
+/datum/controller/subsystem/persistent_data/proc/round_end()
+	Shutdown()
```

Comment:
This is an SS native proc and shouldn't be manually invoked, it could tie into more in the future.

I would much rather the contents of Shutdown() be moved to a dedicated save proc.